io.ossia.score.yml

```diff
@@ -0,0 +1,281 @@
+id: io.ossia.score
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: ossia-score
+rename-desktop-file: ossia-score.desktop
+rename-icon: ossia-score
+finish-args:
+  # BLE support
+  - --allow=bluetooth
+
+  # Access user's data and media libraries, plug-ins etc
+  - --filesystem=host
+
+  # GPU rendering support
+  - --device=dri
+
+  # Required for USB, Webcam devices support as well as raw GPIO I/O
+  - --device=all
+
+  # X11 is used by default
+  - --socket=x11
+
+  # Used by X11
+  - --share=ipc
+
+  # Network access
+  - --share=network
+
+  # PulseAudio is needed for ALSA device access
+  - --socket=pulseaudio
+
+  # Audio support through pipewire
+  - --filesystem=xdg-run/pipewire-0
+
+  # Ignore ALSA configuration
+  - --env=ALSA_CONFIG_PATH=
+
+  # Plugins
+  - --env=VST_PATH=/app/extensions/Plugins/vst
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  - --env=LV2_PATH=/app/extensions/Plugins/lv2
+
+  # Required to allow screensaver/idle inhibition
+  - --talk-name=org.freedesktop.ScreenSaver
+
+add-extensions:
+  # Linux audio plugins base extension
+  org.freedesktop.LinuxAudio.Plugins:
+     directory: extensions/Plugins
+     version: '23.08'
+     add-ld-path: lib
+     merge-dirs: vst;vst3;lv2
+     subdirectories: true
+     no-autodownload: true
+
+  # Full codec support for FFMPEG
+  org.freedesktop.Platform.openh264:
+    directory: lib/openh264,
+    add-ld-path: .,
+    version: "2.4.1"
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg,
+    add-ld-path: .
+    version: "23.08"
+
+cleanup:
+  - '/bin/avahi*'
+  - /include
+  - /share/gir-1.0
+  - /share/man
+  - '*.a'
+  - '*.la'
+  - /lib/cmake
+  - /lib/girepository-1.0
+  - /lib/pkgconfig
+  - /lib/*.la
+  - /lib/*.a
+  - /lib/avahi
+  - /lib/debug
+  - /lib/python*
```

Comment:
Is cleaning up python modules correct?

Reply:
I saw that those were added there by LV2 but I'm pretty sure the app does not use python in any shape or form